src/common/modals/AddInstance/Import.js

```diff
@@ -93,7 +91,16 @@ const Import = ({
         reject(err.stderr);
       });
     });
-    const manifest = await fse.readJson(path.join(tempPath, 'manifest.json'));
+
+    let manifest;
+    try {
+      manifest = await fse.readJson(path.join(tempPath, 'manifest.json'));
+    } catch (err) {
+      setError(true);
+      setLoading(false);
+      console.error(err);
+    }
```

Comment:
When an error will occur, we will still execute the code below. We want to return the function if we encounter an error